docs/drafts/developer.md

```diff
@@ -1,10 +1,24 @@
+
 ## Getting Started
-You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+
+The following `make run` starts a [KIND](https://sigs.k8s.io/kind) cluster for you to get a local cluster for testing, see the manual install steps below for how to run against a remote cluster.
 
 > [!NOTE]
 > Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+> 
+> If you are on MacOS, see [Special Setup for MacOS](#special-setup-for-macos).
```

Comment:
This isn't true with `make run`. `make run`:
1. installs a kind cluster and then updates your kube context to use it.
2. Builds the necessary containers to run operator-controller from the current filesystem.
3. Loads the container into the kind cluster
4. Deploys operator-controller, using the image that was just loaded into the cluster.

Reply:
Not clear which thing you're saying is not true? The first sentence in the NOTE? If so, are you saying delete it? Do you think we should give an overview (like you listed) of what make run is going to do? I think the sentence before the note is likely enough about what make run does.

Reply:
The note isn't true if using `make run`. I'd suggest deleting the sentence about the kube context. I don't think it is necessary to spell out what `make run` does.

Although we may want to have a brief mention about needing `docker` installed and running?

Reply:
ah, okay, swapped in the need for a container runtime environment in https://github.com/operator-framework/operator-controller/pull/1289